src/integration-test/java/se/sundsvall/comfactfacade/apptests/ComfactIT.java

```diff
@@ -22,7 +22,7 @@ class ComfactIT extends AbstractAppTest {
 	void test1_getSigningRequests() {
 
 		setupCall()
-			.withServicePath("/signings")
+			.withServicePath("/signings?page=0&size=1&sort=Created,desc")
```

Comment:
The response seems to have 8 posts but the request says that it only want 1 post per page?

Reply:
Yup. I'll update this to better reflect the responsedata.